src/actions/vstudio/_vstudio.lua

```diff
@@ -271,7 +271,7 @@
 
 		local arch = architecture(cfg.system, cfg.architecture)
 		if not arch then
-			arch = iif(isnative, "x86", "Any CPU")
+			arch = iif(isnative, "x86", "AnyCPU")
```

Comment:
I'm not exactly sure what the issue you're trying to resolve is, but this has introduced another bug, the configuration is now `Debug Any CPU` instead of `Debug`.